src/main/java/Storage.java

```diff
@@ -0,0 +1,66 @@
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+/**
+ * The Storage class deals with loading tasks from the file and saving tasks in the file.
+ */
+public class Storage {
+    TaskList taskList = new TaskList();
+    private static File file;
+
+    /**
+     * Constructor for Storage class.
+     */
+    public Storage() {
+        file = new File("./data/duke.txt");
+    }
+
+    /**
+     * Loads tasks stored in duke.txt in the hard disk. Creates and add the tasks into the ArrayList of tasks.
+     * @throws IOException Throws IOException.
+     * @throws DukeException Throws DukeException.
+     */
+    public void loadData() throws IOException, DukeException {
+        // load data from ./data/duke.txt
+        file.createNewFile();
+
+        BufferedReader br = new BufferedReader(new FileReader(file));
+        String str;
+
+        while ((str = br.readLine()) != null) {
```

Comment:
This is a rather unconventional statement.
I would recommend adjusting the code.

Perhaps:

On line 30:     str = br.readLine();
On line 31:    while (str != null) {
On line 45:   str = br.readLine();